services/download/src/main/resources/application.yaml

```
@@ -19,6 +19,7 @@ services:
     retention-days: 14
     validation:
       allowed-report-types: ${ALLOWED_REPORT_TYPES:CONFIRMED_TEST,CONFIRMED_CLINICAL_DIAGNOSIS}
+    batch-audit-enabled: ${EFGS_ENABLE_BATCH_AUDIT:false}
```

can we please keep environment variables names as similar as possible?

```suggestion
    batch-audit-enabled: ${BATCH_AUDIT_ENABLED:false}
```